ArtOfIllusion/src/artofillusion/RotateViewTool.java

```diff
@@ -154,18 +157,18 @@ private void dragRotateTravelSpace(WidgetMouseEvent e, ViewerCanvas view)
 		dx = dragPoint.x-clickPoint.x;
 		dy = dragPoint.y-clickPoint.y;
 
-		if (controlDown && !e.isShiftDown())
+		if (mouseButtonTwo(e) && controlDown)
 		{
 			view.tilting = true;
 			tilt(e, view, clickPoint);
 			return;
 		}
-		else if (controlDown && e.isShiftDown())
+		else if (mouseButtonTwo(e))
```

Comment:
This is supposed the temporary switce between normal rotate and pan. usin bpths Shift and Control. MouseButtonTwo alone has a different function.

```suggestion
 		else if (e.isControlDown() && e.isShiftDown())
```